ext/date/php_date.c

```diff
@@ -4952,6 +4952,11 @@ PHP_METHOD(DatePeriod, __construct)
 	dpobj->current = NULL;
 
 	if (isostr) {
+		zend_error(E_DEPRECATED, "Calling DatePeriod::__construct(string $isostr, int $options = 0) is deprecated");
```

Comment:
I would add the suggested call into the deprecation messages.
```suggestion
		zend_error(E_DEPRECATED, "Calling DatePeriod::__construct(string $isostr, int $options = 0) is deprecated, "
			"use DatePeriod::createFromISO8601String() instead");
```

Reply:
OK, I added the suggested alternatives to the error messages.